MobileWallet/Common/Theme.swift

```diff
@@ -117,6 +117,10 @@ struct Colors: Loopable {
     let transactionCellValuePositiveText = UIColor(named: "TransactionCellValuePositiveText")
     let transactionSmallSubheadingLabel = UIColor(named: "SmallSubheading")
 
+    // Sending Tari
+    let sendingTariTitle = UIColor(named: "SendingTariTitleTextBlack")
```

Comment:
Call this `SendingTariTitleText` instead of `SendingTariTitleTextBlack`. No need to include the color in the filename as the color might change at a later stage, especially when we switch to darkmode support

Reply:
Done.